<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="myLabel" name="myScreen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
alternate-background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="showAxes">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gridSize">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="showGrid">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>270</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Grid</string>
    </property>
   </widget>
   <widget class="QLabel" name="gridSizeLabel">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Size</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>80</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mouseMovementLabel">
       <property name="text">
        <string>Mouse Movement</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mouseMovement">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mousePressedLabel">
       <property name="text">
        <string>Mouse Pressed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mousePressed">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="setPoint1">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Point 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setPoint2">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Point 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineDrawingLabel">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Line Drawing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectLineDrawAlgo">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>540</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DDA Line Drawing Alogrithm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Breshenham Line Drawing Algorithm</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="unsetPoint2">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>440</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unset Point 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unsetPoint1">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>440</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unset Point 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectLineDrawAlgoLabel">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Line Drawing Algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawLine">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>590</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Line</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header location="global">myLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
